problemsets-mat223/linearalbegra.tex

```diff
@@ -2288,14 +2313,107 @@ \section*{Subspaces and Bases}
+			\begin{solution}
+				XXXX
+					
+				$C$ is a subspace.
+
+				\begin{enumerate}[label=(\roman*)]
+					\item Let $\vec u,\vec v\in C$. Then $\vec u=\mat{0\\t}$ and
+						$\vec v=\mat{0\\s}$ for some $s,t\in\R$.
+
+						But then $\vec u+\vec v=\mat{0\\s+t}\in C$. 
+					\item Let $\vec u=\mat{0\\t}\in C$ and let $\alpha$ be any scalar.
+						Then $\alpha\vec u=\mat{0\\\alpha t}\in C$. 
+				\end{enumerate}
+			\end{solution}
+		\item $D=\Set*{\vec x\in\R^2 \given \vec x=\mat{0\\t}+\mat{1\\1}\text{ for some }t\in\R}$.
+			\begin{solution}
+				XXXX
+					
+				$D$ is not a subspace, since for example $\mat{1\\1}\in D$, but
+				$0\mat{1\\1}=\mat{0\\0}\notin D$. 
+			\end{solution}
+		\item $E=\Set*{\vec x\in\R^2 \given \vec x=\mat{0\\t}\text{ or }\vec x=\mat{t\\0}\text{ for some }t\in\R}$.
+			\begin{solution}
+				XXXX
+
+				$E$ is not a subspace, since for example  $\mat{1\\0}$ and $\mat{0\\1}$
+				are both in $E$, but their sum is $\mat{1\\1}$ which is not in $E$.
+			\end{solution}
+		\item $F=\Set*{\vec x\in\R^2 \given \vec x=t\mat{3\\1}\text{ for some }t\in\R}$.
+			\begin{solution}
+				XXXX
+					
+				$F$ is a subspace.
+
+				\begin{enumerate}[label=(\roman*)]
+					\item Let $\vec u,\vec v\in F$. Then $\vec u=t\mat{3\\1}$ and
+						$\vec v=s\mat{3\\1}$ for some $s,t\in\R$. 
+
+						But then $\vec u+\vec v=(s+t)\mat{3\\1}\in F$. 
+					\item Let $\vec u=t\mat{3\\1}\in F$ and let $\alpha$ be any scalar.
+						Then $\alpha\vec u=(\alpha t)\mat{3\\1}\in F$. 
+				\end{enumerate}
+			\end{solution}
 		\item $G=\Span\left\{\mat{1\\1}\right\}$.
-		\item $H=\Span\{\vec u,\vec v\}$ for some unknown vectors $\vec u,\vec v\in\R^2$.
+			\begin{solution}
+				XXXX
+					
+				$G$ is a subspace. 
+
+				By definition of a span,
+				$G=\Set*{\vec x\in\R^2 \given \vec x=t\mat{1\\1}\text{ for some }t\in\R}$.
+
+				The proof that $G$ is a subspace now proceeds similarly to the
+				proof for $F$ above.
+
+				\begin{enumerate}[label=(\roman*)]
+					\item Let $\vec u,\vec v\in G$. Then $\vec u=t\mat{1\\1}$ and
+						$\vec v=s\mat{1\\1}$ for some $s,t\in\R$. 
+
+						But then $\vec u+\vec v=(s+t)\mat{1\\1}\in G$. 
+					\item Let $\vec u=t\mat{1\\1}\in G$ and let $\alpha$ be any scalar.
+						Then $\alpha\vec u=(\alpha t)\mat{1\\1}\in G$. 
+				\end{enumerate}
+
+		\end{solution}
+		\item $H=\Span\Set{\vec u,\vec v}$ for some unknown vectors $\vec u,\vec v\in\R^2$.
+			\begin{solution}
+				XXXX
+					
+				$H$ is a subspace. 
+
+				\begin{enumerate}[label=(\roman*)]
+					\item Let $\vec x,\vec y\in H$.
+						Then $\vec x=\alpha_1\vec u+\alpha_2\vec v$ and 
+						$\vec y=\beta_1\vec u+\beta_2\vec v$ for some scalars 
+						$\alpha_1,\alpha_2,\beta_1,\beta_2$. But then
+						\[
+							\vec x+\vec y
+							=\alpha_1\vec u+\alpha_2\vec v+\beta_1\vec u+\beta_2\vec v
+							=(\alpha_1+\beta_1)\vec u+(\alpha_2+\beta_2)\vec v\in H.
+						\]
+
+					\item Let $\vec x=\alpha_1\vec u+\alpha_2\vec v\in H$ and let $\beta$ be any scalar.
```

Comment:
We cannot let `\beta` be any scalar (it can only be one scalar...). Rephrase as: "For any scalar \beta..."